File: 27-remove-element/remove-element.ts
function removeElement(nums: number[], val: number): number {
    let slow = 0, i = 0;
    while (i<nums.length) {
        if(nums[i] !=val){
            nums[slow]=nums[i]
            slow+=1
        }i++
    }
    // let k=nums.length
    return slow
};

File: 27-remove-element/remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        arr = []
        for i in range(len(nums)):
            if nums[i] != val:
                arr.append(nums[i])
        for i in range(len(arr)):
            nums[i] = arr[i]

        return len(arr)

